### PR TITLE
adding hack/build* to make life easy and added version command

### PR DESCRIPTION
**Pull request by maximilien**
Getting my feet wet...

### CODE REVIEW

File: pkg/kn/commands/version.go [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+package commands
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+)
+
+const (
+	Version = "0.0.0"
```
Comment (rhuss) [inline pkg/kn/commands/version.go]: How is this version set ? Imo this should go into the build script via `-ldflags "-X commands.Version=0.1.0"` to `go build`.

Comment (maximilien) [reply pkg/kn/commands/version.go]: Yeah, we can do that. I'll add it. 

FYI... Still trying to get the CLA thingie done.

**Comment by maximilien** [comment]
I signed it... I think

**Comment by sixolet** [comment]
/approve
Waiting on lgtm for response to rhuss

**Comment by maximilien** [comment]
Hi, @sixolet, thanks. The `-ldflags "-X commands.Version=0.1.0` did not work for me. Silently works but no version information is passed so the Version variable is set to default "" (empty string).

I investigated a bit and trying new Golang versions to see if that makes a difference. Will try a couple more things, if still not working I'll push the code so you all can see what I mean.

**Pull request merged**